ms/MSOper/MSMetaData.h

```diff
@@ -85,12 +85,13 @@ class MSMetaData {
         // the subscan and that spwID
         map<uInt, Quantity> meanInterval;
         Quantity meanExposureTime;
-        uInt nrows;
         std::set<uInt> spws;
         // number of rows for each spectral window
         map<uInt, uInt> spwNRows;
         std::set<Int> stateIDs;
-        std::map<Double, TimeStampProperties> timeProps;
+        map<Double, TimeStampProperties> timeProps;
```

Comment:
This is the first I'm hearing about this. Why is it not permitted to use using namespace std; ?